Source/Data/Local/Encrypted/ParleyEncryptedDataSource.swift

```diff
@@ -97,24 +97,27 @@ extension ParleyEncryptedDataSource: ParleyKeyValueDataSource {
 extension ParleyEncryptedDataSource: ParleyMessageDataSource {
     
     public func all() -> [Message]? {
-        if let jsonString = self.string(forKey: kParleyCacheKeyMessages), let messages: [Message] = Array(JSONString: jsonString) {
-            messages.forEach { message in
-                if message.type == .user, message.status == .pending, let uuid = message.uuid, let imageData = self.data(forKey: uuid), let imageUrl = message.imageURL {
-                    message.imageData = imageData
-                    
-                    switch ParleyImageType.map(from: imageUrl) {
-                    case .gif:
-                        message.image = UIImage.gif(data: imageData)
-                    default:
-                        message.image = UIImage(data: imageData)
-                    }
+        guard let jsonData = self.data(forKey: kParleyCacheKeyMessages),
+              let messages = try? CodableHelper.shared.decode([Message].self, from: jsonData)
```

Comment:
The `Array(JSONString: jsonString)` did also return nil if the mapping failed. So I've done the same (ignoring the error).